lib/commands/travis-visual-acceptance.js

```diff
@@ -91,11 +100,14 @@ module.exports = {
     process.env.NO_ASSERT = true
 
     var prNumber = process.env.TRAVIS_PULL_REQUEST
+
     if (prNumber !== false && prNumber !== 'false' && process.env.VISUAL_ACCEPTANCE_TOKEN) {
       return runCommand(this, 'br').then(buildReport, function (params) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then

Currently ur only handling `onRejected` (`ember test` failed`). You need to handle `onPassed` with  
```
if (options.msgOn === options.msgVar) {
           return buildReport(params)
        }
```
as well